lib/Service/AccountService.php

```diff
@@ -233,9 +233,14 @@ public function getConfig(?IUser $user = null): array {
 		$info['hasSignatureFile'] = $this->hasSignatureFile($user);
 		$info['phoneNumber'] = $this->getPhoneNumber($user);
 		$info['isApprover'] = $this->validateHelper->userCanApproveValidationDocuments($user, false);
+		//$info['grid_view'] = $this->appConfig->getValueBool(Application::APP_ID, 'grid_view', false);
+		$info['grid_view'] = $this->config->getUserValue($user->getUID(), Application::APP_ID, 'grid_view', false) === '1';
```

Comment:
Check the integration tests and the type of $user, could be null when is unauthenticated signer.